.editorconfig

```diff
@@ -0,0 +1,12 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# 2 space indentation
+[*.{js,json}]
+indent_style = space
+indent_size = 2
```

Comment:
Can you add a new line at the end of the file? 👍
Other than that it's ready to go!